WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerce.xcscheme

```diff
@@ -120,8 +120,8 @@
       </BuildableProductRunnable>
       <CommandLineArguments>
          <CommandLineArgument
+            argument = ""
             isEnabled = "NO">
-            argument = "mocked-network-layer"
          </CommandLineArgument>
```

Comment:
This looks like an unintentional change. This is now empty in the scheme. This does not seem to affect the screenshots tests though. I think it's just used for the development of the Screenshots package. 

<img src="https://user-images.githubusercontent.com/198826/107263355-dfa9f800-69fe-11eb-8543-37f0e8494ced.png" width="300">

Reply:
Interesting. I'll revert this. Thanks!